build.gradle

```diff
@@ -255,6 +256,21 @@ subprojects {
       }
     }
 
+    slf4jApi = { ->
```

Comment:
Looks that we don't even need this as a separate function, you can just inline it at the single call site below inside `slf4jLambdaApi`

Reply:
Yes, I can but IMHO it doesn't bother us and it's more flexible for future usages when we have this separated.